### PR TITLE
Frontload BlockMediator (bmd8r)

### PR DESCRIPTION
**Pull request by JingleH**
Attach BlockMediator (bmd8r) to window in scripts.js, instead of relying on a dynamic network call to fetch when needed, as it's already heavily depended upon for most pages. This will also make mocking it in unit testing a lot easier

Resolves: https://jira.corp.adobe.com/browse/MWPW-143054

Test URLs:
- Before: https://stage--express--adobecom.hlx.page/express/pricing?martech=off
- After: https://front-load-block-mediator--express--adobecom.hlx.page/express/pricing?martech=off

### CODE REVIEW

File: express/scripts/scripts.js [hunk 1]
```diff
@@ -121,6 +121,9 @@ const listenAlloy = () => {
   }, 5000);
 };
 
+// eslint-disable-next-line
+window.bmd8r=(()=>{const r={};const i=s=>{r[s]={cbs:[]}};const b=s=>s in r;const s=()=>Object.keys(r);const t=s=>r[s]?.v;const c=(n,o)=>{if(!b(n)){i(n)}const e=t(n);r[n].v=o;return new Promise(s=>{const t=[];for(const c of r[n].cbs){try{c({oldValue:e,newValue:o})}catch(s){t.push(s)}}s(t)})};const n=(s,t)=>{if(!b(s)){i(s)}const c=r[s];if(c.cbs.includes(t))return()=>{};c.cbs.push(t);const n=()=>{c.cbs=c.cbs.filter(s=>s!==t)};return n};return{hasStore:b,listStores:s,get:t,set:c,subscribe:n}})();
```
[the remark this answers is not in the document]
Comment (JingleH) [reply express/scripts/scripts.js:125]: we can indeed... but I tried that and it seems to be making our scripts.js feel longer than it is 😢 let's see if people prefer the long one

[the remark this answers is not in the document]
Comment (JingleH) [reply express/scripts/scripts.js:125]: it's only about 500 characters, but it takes 50 lines...

[the remark this answers is not in the document]
Comment (JingleH) [reply express/scripts/scripts.js:125]: @vhargrave what would be your thoughts on this? You can try copying the unminified version block-mediator.js into scripts.js, remove comments and a few newLines, and check if it's acceptable

[the remark this answers is not in the document]
Comment (JingleH) [reply express/scripts/scripts.js:125]: I kinda wanted this thing to be as inconspicuous as possible

[the remark this answers is not in the document]
Comment (qiyundai) [reply express/scripts/scripts.js:125]: I see! I'm OK with either. Let's see what people say about it!

**Review by qiyundai — APPROVED** [review]

**Pull request closed without merging**